src/manifolds/GraphManifold.jl

```diff
@@ -21,7 +21,7 @@ A type for a [`GraphManifold`](@ref) where the data is given on the vertices.
 struct VertexManifold <: GraphManifoldType end
 
 @doc doc"""
-    GraphManifold{G, M, T} <: AbstractPowerManifold{M}
+    GraphManifold{G, M, T} <: AbstractPowerManifold{M,,NestedPowerRepresentation}
```

Comment:
Can we document here or in ? `graph.md` how one has to create such a nested data? That would be helpful to the user.

Reply:
Sure, I'll add an example to `graph.md`.